CHANGE-NOTES.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [Unreleased]
+
+- Show only VCS error notification when machete file parsing fails (no IDE internal error).
```

Comment:
Hmm this is coming from some the older releases (2.1.0 or so).

What you can do though is to fill up the changelog with the older releases, based either on GitHub Releases or `git show <tag>:CHANGE-NOTES.html` (should be almost equivalent)